crates/core/tedge_api/src/mqtt_topics.rs

```diff
@@ -602,6 +604,9 @@ pub enum Channel {
         operation: OperationType,
         cmd_id: String,
     },
+    Signal {
+        signal_type: SignalType,
```

Comment:
Using a regular `String` would work as well.

Reply:
Yes, but we will lose the enum representation in the code. I prefer to keep the enum to track easily what we support.